docs/ArrowKeyStepper.md

```diff
@@ -1,7 +1,7 @@
 ArrowKeyStepper
 ---------------
 
-High-order component that decorates another virtualized component and responds to arrow-key events by scrolling one row or column at a time.
+A High-order component (A component that takes a component and returns a component) that decorates another virtualized component and responds to arrow-key events by scrolling one row or column at a time.
```

Comment:
I'd say it's not higher order component. Can you call it `A component with children as function that wraps...`?

Reply:
Yeah, I guess these docs are a little out of date. `ArrowKeyStepper` used to be implemented more as a HOC but changed to use the function children / render props a long time ago. Maybe the docs could be updated to say something more like...

> Decorates another react-virtualized component such as `Grid` or `List`. Passes `scrollToColumn` and `scrollToRow` props to a render function to enable user to navigate content using arrow keys.